src/frontends/tensorflow/src/tensorflow.cpp

```diff
@@ -2,6 +2,8 @@
 // SPDX-License-Identifier: Apache-2.0
 //
 
+#include <google/protobuf/stubs/logging.h>
+
```

Comment:
please do not include protobuf dependencies here. Can we face segfault issue on OVTF side, if they use different protobuf version.

Reply:
Why? it is our internal code. We use static protobuf inside our components

Reply:
the problem is explained below